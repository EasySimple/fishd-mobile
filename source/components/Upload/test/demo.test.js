import webDemoTest from '../../../tests/demoTest';

webDemoTest('Upload');
