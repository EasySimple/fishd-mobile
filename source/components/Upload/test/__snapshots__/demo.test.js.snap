// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders Upload demo correctly 1`] = `
<div>
  <div
    class="components-tpl-demo-basic"
  >
    <div
      class="demo-title"
    >
      基础用法
    </div>
    <div
      class="demo-card"
    >
      <div
        class="fm-upload"
      >
        <div
          class="fm-upload__wrapper"
        >
          <div
            class="fm-upload__upload"
          >
            <i
              class="icon iconfont fm-icon fm-icon-fm-add"
              style="width:30px;height:30px;color:#D8D8D8"
            />
            <input
              accept="image/*"
              class="fm-upload__input"
              type="file"
            />
          </div>
        </div>
      </div>
    </div>
  </div>
  <div
    class="components-tpl-demo-basic"
  >
    <div
      class="demo-title"
    >
      可预览
    </div>
    <div
      class="demo-card"
    >
      <div
        class="fm-upload"
      >
        <div
          class="fm-upload__wrapper"
        >
          <div
            class="fm-upload__preview"
          >
            <div
              class="fm-upload__image"
            >
              <img
                src="https://placekitten.com/600/400"
                style="object-fit:cover"
              />
            </div>
            <div
              class="fm-upload__delete"
            >
              <i
                class="icon iconfont fm-icon fm-icon-fm-close"
                style="width:10px;height:10px;color:#fff"
              />
            </div>
          </div>
          <div
            class="fm-upload__preview"
          >
            <div
              class="fm-upload__image"
            >
              <img
                src="https://placekitten.com/1200/900"
                style="object-fit:cover"
              />
            </div>
            <div
              class="fm-upload__delete"
            >
              <i
                class="icon iconfont fm-icon fm-icon-fm-close"
                style="width:10px;height:10px;color:#fff"
              />
            </div>
          </div>
          <div
            class="fm-upload__upload"
          >
            <i
              class="icon iconfont fm-icon fm-icon-fm-add"
              style="width:30px;height:30px;color:#D8D8D8"
            />
            <input
              accept="image/*"
              class="fm-upload__input"
              type="file"
            />
          </div>
        </div>
      </div>
    </div>
  </div>
  <div
    class="components-tpl-demo-basic"
  >
    <div
      class="demo-title"
    >
      上传状态
    </div>
    <div
      class="demo-card"
    >
      <div
        class="fm-upload"
      >
        <div
          class="fm-upload__wrapper"
        >
          <div
            class="fm-upload__preview"
          >
            <div
              class="fm-upload__image"
            >
              <img
                src="https://placekitten.com/600/400"
                style="object-fit:cover"
              />
            </div>
            <div
              class="fm-upload__mask"
            >
              <div
                class="fm-loadmore fm-loadmore__normal"
              >
                <span
                  class="fm-loadmore__spinner fm-loadmore__spinner--circular"
                  style="color:#fff;width:30px;height:30px"
                >
                  <svg
                    class="fm-loadmore__circular"
                    viewBox="25 25 50 50"
                  >
                    <circle
                      cx="50"
                      cy="50"
                      fill="none"
                      r="20"
                    />
                  </svg>
                </span>
              </div>
              <div
                class="fm-upload__mask-message"
              >
                上传中...
              </div>
            </div>
            <div
              class="fm-upload__delete"
            >
              <i
                class="icon iconfont fm-icon fm-icon-fm-close"
                style="width:10px;height:10px;color:#fff"
              />
            </div>
          </div>
          <div
            class="fm-upload__preview"
          >
            <div
              class="fm-upload__image"
            >
              <img
                src="https://placekitten.com/1200/900"
                style="object-fit:cover"
              />
            </div>
            <div
              class="fm-upload__mask"
            >
              <i
                class="icon iconfont fm-icon fm-icon-fm-error"
                style="width:30px;height:30px;color:#fff"
              />
              <div
                class="fm-upload__mask-message"
              >
                上传失败
              </div>
            </div>
            <div
              class="fm-upload__delete"
            >
              <i
                class="icon iconfont fm-icon fm-icon-fm-close"
                style="width:10px;height:10px;color:#fff"
              />
            </div>
          </div>
          <div
            class="fm-upload__upload"
          >
            <i
              class="icon iconfont fm-icon fm-icon-fm-add"
              style="width:30px;height:30px;color:#D8D8D8"
            />
            <input
              accept="image/*"
              class="fm-upload__input"
              type="file"
            />
          </div>
        </div>
      </div>
    </div>
  </div>
  <div
    class="components-tpl-demo-basic"
  >
    <div
      class="demo-title"
    >
      限制上传数量
    </div>
    <div
      class="demo-card"
    >
      <div
        class="fm-upload"
      >
        <div
          class="fm-upload__wrapper"
        >
          <div
            class="fm-upload__preview"
          >
            <div
              class="fm-upload__image"
            >
              <img
                src="https://placekitten.com/600/400"
                style="object-fit:cover"
              />
            </div>
            <div
              class="fm-upload__delete"
            >
              <i
                class="icon iconfont fm-icon fm-icon-fm-close"
                style="width:10px;height:10px;color:#fff"
              />
            </div>
          </div>
          <div
            class="fm-upload__upload"
          >
            <i
              class="icon iconfont fm-icon fm-icon-fm-add"
              style="width:30px;height:30px;color:#D8D8D8"
            />
            <input
              accept="image/*"
              class="fm-upload__input"
              type="file"
            />
          </div>
        </div>
      </div>
    </div>
  </div>
  <div
    class="components-tpl-demo-basic"
  >
    <div
      class="demo-title"
    >
      限制上传大小
    </div>
    <div
      class="demo-card"
    >
      <div
        class="fm-upload"
      >
        <div
          class="fm-upload__wrapper"
        >
          <div
            class="fm-upload__preview"
          >
            <div
              class="fm-upload__image"
            >
              <img
                src="https://placekitten.com/600/400"
                style="object-fit:cover"
              />
            </div>
            <div
              class="fm-upload__delete"
            >
              <i
                class="icon iconfont fm-icon fm-icon-fm-close"
                style="width:10px;height:10px;color:#fff"
              />
            </div>
          </div>
          <div
            class="fm-upload__upload"
          >
            <i
              class="icon iconfont fm-icon fm-icon-fm-add"
              style="width:30px;height:30px;color:#D8D8D8"
            />
            <input
              accept="image/*"
              class="fm-upload__input"
              type="file"
            />
          </div>
        </div>
      </div>
    </div>
  </div>
  <div
    class="components-tpl-demo-basic"
  >
    <div
      class="demo-title"
    >
      禁用上传
    </div>
    <div
      class="demo-card"
    >
      <div
        class="fm-upload"
      >
        <div
          class="fm-upload__wrapper"
        >
          <div
            class="fm-upload__upload fm-upload__disabled"
          >
            <i
              class="icon iconfont fm-icon fm-icon-fm-add"
              style="width:30px;height:30px;color:#D8D8D8"
            />
            <input
              accept="image/*"
              class="fm-upload__input"
              disabled=""
              type="file"
            />
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
